config/database.yml

```diff
@@ -20,8 +20,11 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: <%= ENV['USERNAME'] %>
-  password: <%= ENV['PASSWORD'] %>
+  #username: <%= ENV['USERNAME'] %>
+  #password: <%= ENV['PASSWORD'] %>
+  host: 127.0.0.1
+  username: postgres
+  password: K@i14969
```

Comment:
Maybe we need to keep environment variables and only modify our .env files in each local repository; otherwise every time we pull dev brach we will have to change this file again and again